docs/ENVIRONMENT.md

```diff
@@ -254,6 +254,14 @@ systemd itself:
   it is either set to `system` or `user` depending on whether the NSS/PAM
   module is called by systemd in `--system` or `--user` mode.
 
+* `$SYSTEMD_EXEC_PID=…` — set for all invocations that are done by the service
+  manager on behalf of a specific unit, in child processes that are later
+  (after execve()) going to become unit processes (e.g. process invoked by
+  ExecStart=). Contains the process ID (PID) of the child process. The child
+  process can use this information to determine whether the process is directly
+  invoked by the service manager or indirectly through a script by comparing
+  this value with the current process ID.
```

Comment:
This should be moved to "Environment Variables Set or Propagated by the Service Manager" in systemd.exec since it'll be visible in the env block of spawned processes.

Strictly speaking, a binary (not a script), may also invoke another binary. So maybe change this to
> is directly invoked by the service manager or indirectly as a child of another process

Also I'd add:
> (This is similar to the scheme with $LISTEN_FDS and $LISTEN_PID.)